src-backbone/app/js/views/auth/resetPasswordView.js

```diff
@@ -0,0 +1,63 @@
+let App = require('utils/sanaAppInstance');
+const Helpers = require('utils/helpers');
+
+module.exports = Marionette.ItemView.extend({
+    template: require('templates/auth/resetPasswordView'),
+
+    ui: {
+        form: 'form',
+    },
+
+    events: {
+        'submit': 'onSubmit',
```

Comment:
I'm more surprised that this event even triggered. I did not know the form event can propagate up the DOM tree. 

Should change it to `@ui.form submit`